src/renderer/components/+extensions/install.tsx

```diff
@@ -30,17 +28,22 @@ interface Dependencies {
   extensionInstallationStateStore: ExtensionInstallationStateStore;
 }
 
-const installInputValidators = [
-  InputValidators.isUrl,
-  InputValidators.isPath,
-  InputValidators.isExtensionNameInstall,
-];
+const installInputValidator = asyncInputValidator({
+  validate: async (value) => {
+    if (
+      InputValidators.isUrl.validate(value)
+      || InputValidators.isExtensionNameInstall.validate(value)
+    ) {
+      return;
```

Comment:
Do we need to return `true`? A bit confusing for me..

Reply:
Fair enough, the rule is that synchronous validators must return booleans but async validators reject to show errors.